src/components/MapView/index.js

```diff
@@ -14,8 +14,33 @@ const MapView = ({ locations }) => {
   });
 
   const unknown = [];
+  const markers = [];
+
+  // maps coordinates to the locations at that coordinate
+  const coordsToLocationMap = new Map();
   locations.forEach((location) => {
-    if (!location.coords) unknown.push(location);
+    if (!location.coords) {
+      unknown.push(location);
+    } else if (coordsToLocationMap.has(location.coords)) {
+      coordsToLocationMap.get(location.coords).push(location);
+    } else {
+      coordsToLocationMap.set(location.coords, [location]);
+    }
+  });
+
+  coordsToLocationMap.forEach((coordLocations, coords) => {
+    markers.push(
+      <Marker latitude={coords.lat} longitude={coords.long}>
```

Comment:
`<Marker>` here should have a `key` prop: https://reactjs.org/docs/lists-and-keys.html (I'd recommend just using the index in the list since these items won't change at runtime)